Gems/Atom/Feature/Common/Assets/Materials/Types/NewPBR/NewPBR_Surface.azsli

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <Atom/Features/MatrixUtility.azsli>
```

Comment:
It would be nice to #include all the MaterialInput files here directly rather than requiring them to be stitched in.